packages/react-storage/src/components/StorageBrowser/controls/hooks/usePagination.ts

```diff
@@ -0,0 +1,31 @@
+import { PaginationProps } from '../../composables/Pagination';
+import { useControlsContext } from '../context';
+
+export type UsePagination = () => {
+  props?: PaginationProps;
+};
```

Comment:
As a result of the below comment, we can just get rid of this type

Reply:
Done